src/lib/services/token/getGreenBeanStatus.ts

```diff
@@ -0,0 +1,25 @@
+import { Collection } from "@/lib/emblems/enums";
+import { GreenBeanStatus, Nft } from "./types";
+import ky from "ky";
+import { getBaseUrl } from "@/utils/getBaseUrl";
+
+export async function getGreenBeanStatus(collections: {
+  tokens: Nft[];
+}): Promise<GreenBeanStatus> {
+  if (collections.tokens[0].token.contract === Collection.AZUKI) {
```

Comment:
i believe it always returns false, given the uppercase/lowercase not matching.. but what is this check trying to do? :)

Reply:
Actually, it works correctly for Azuki's. I added this check to avoid requesting the RPC with Elementals or Beanz

Reply:
yea thats my thought too - i recalled when i wrote code earlier, this comparison won't work since the address in both are not matching uppercase/lowercase with a direct string comparison.. anyway i can confirm it later, lgtm for now ^^